### PR TITLE
[FE] creat (molecule) : Search component

### PR DESCRIPTION
**Pull request by WilkerAtalaya**
create componete AND test it

### CODE REVIEW

File: frontend/src/components/atoms/SearchComponents/titleSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const TitleSearchWithImage = () => {
```
Comment (AnthonyRicse98) [inline frontend/src/components/atoms/SearchComponents/titleSearch.tsx:3]: the name is very long , use WSearch

File: frontend/src/components/atoms/SearchComponents/titleSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      };
+
+  return (
+    <div style={containerStyle}>
+      <img
+        src="/images/FrameStudentNET.png" 
+        alt="FrameStudentNET"
+        style={imageStyle}
+      />
+      <h1>Student Network</h1>
```
Comment (AnthonyRicse98) [inline frontend/src/components/atoms/SearchComponents/titleSearch.tsx:24]: use props the name of Student Network is a textfield, look for a search component in material ui or something similar

File: frontend/src/components/atoms/SearchComponents/titleSearch.tsx
Comment (AnthonyRicse98) [inline frontend/src/components/atoms/SearchComponents/titleSearch.tsx:1]: the file es molecule , it is not an atom

File: frontend/src/app/auth/login/page.tsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
 'use client'
 import EnrollmentHoc from "@/app/auth/auth";
 import CardAuth from "@/components/organisms/CardAuth";
+import LogoTitle from "@/components/atoms/SearchComponents/titleSearch";
```
Comment (AnthonyRicse98) [inline frontend/src/app/auth/login/page.tsx:4]: use in page test  , don't try it on the login page 
use localhost:3000/test

**Review by eds000n — COMMENTED** [review]
please fix conflicts

File: frontend/package.json [hunk 4]
```diff
@@ -13,12 +13,13 @@
...
     "@emotion/styled": "^11.11.0",
-    "@mui/icons-material": "^5.14.9",
-    "@mui/material": "^5.14.9",
+    "@mui/icons-material": "^5.14.11",
+    "@mui/material": "^5.14.11",
     "@types/node": "20.6.0",
     "@types/react": "18.2.21",
     "@types/react-dom": "18.2.7",
     "axios": "^1.5.0",
+    "i": "^0.3.7",
```
Comment (AnthonyRicse98) [inline frontend/package.json:22]: delete line package json

**Review by RicseDev — APPROVED** [review]

**Review by AnthonyRicse98 — DISMISSED** [review]

**Review by AnthonyRicse98 — APPROVED** [review]

**Pull request merged**